core/src/main/java/org/axonframework/commandhandling/model/AggregateLifecycle.java

```diff
@@ -130,16 +130,13 @@ public static void markDeleted() {
      */
     protected static AggregateLifecycle getInstance() {
         AggregateLifecycle instance = Scope.getCurrentScope();
-        if (instance == null && CurrentUnitOfWork.isStarted()) {
+        if (CurrentUnitOfWork.isStarted()) {
```

Comment:
Don't think this is semantically correct anymore. The lookup of an AggregateInstance in the UnitOfWork should only be done if there isn't a Current Scope defined. If the Current Scope is an aggregate, that should always take precedence.